locations/spiders/5asec.py

```diff
@@ -0,0 +1,32 @@
+import re
+from typing import Any
+
+from scrapy.http import Response
+from scrapy.spiders import SitemapSpider
+
+from locations.items import Feature
+
+
+class FiveAsecSpider(SitemapSpider):
+    name = "5asec"
```

Comment:
CI says class name should be `5AsecSpider`, but it will not be valid. I suggest to change spider name and file name to `five_asec`.

```suggestion
    name = "five_asec"
```

(Or it can be left as is and force-merged as we already have similar spiders in the codebase)